registry.json

```
@@ -50,6 +50,17 @@
     },
     "contributors": []
   },
+  {
+    "name": "mac-address",
+    "nestdotland": "https://nest.land/package/mac-address",
```

```suggestion
    "nestdotland": "mac-address",
```

all module names will need to be formatted this way 😅

---

@t8 sorry for the late response --> corrected it for all just now - if I get it correctly :)